raiden/api/v1/resources.py

```diff
@@ -213,7 +213,7 @@ def get(self):
 class PaymentResource(BaseResource):
 
     post_schema = PaymentSchema(
-        only=('amount', 'identifier'),
+        only=('amount', 'identifier', 'secret', 'secret_hash'),
```

Comment:
The `secret` should not be exposed in the return response of the API call.

Reply:
It should per https://github.com/raiden-network/raiden/issues/3317#issue-402631441

Reply:
The way i was thinking about this is that the secret is shared **before** the payment is complete. But i realized that the response is returned **after** the payment is complete so i think this should be fine.